### ISSUE TITLE
LowerValueChanged and UpperValueChanged events are not fired on Android

### ISSUE THREAD

**Issue by Zolee007** [halkar/xamarin-range-slider#18]
`LowerValueChanged` and `UpperValueChanged` events are not fired, if `NotifyWhileDragging` is `false`.
The probable cause of it relies in `OnTouchEvent`'s `MotionEventActions.Up` case, where `_pressedThumb` is set to `null`, before the checks on events, so they will be never raised.

The mentioned code piece:
```
_pressedThumb = null;
Invalidate();
if (_pressedThumb == Thumb.Min)
    OnLowerValueChanged();
if (_pressedThumb == Thumb.Max)
    OnUpperValueChanged();
```

If possible only set `_pressedThumb` to `null` after the events where checked and fired.

**Comment by halkar** [comment]
Thanks @Zolee007, will investigate.

**Comment by halkar** [comment]
@Zolee007 issue should be fixed in version 0.6.2.

**Comment by Zolee007** [comment]
@halkar You are awesome, thank you.
